pkgs/development/web/bun/default.nix

```diff
@@ -33,7 +33,7 @@ stdenvNoCC.mkDerivation rec {
     runHook postInstall
   '';
 
-  postPhases = [ "postPatchelf" ];
+  postPhases = lib.optionals (stdenvNoCC.buildPlatform.canExecute stdenvNoCC.hostPlatform) [ "postPatchelf" ];
   postPatchelf = lib.optionalString (stdenvNoCC.buildPlatform.canExecute stdenvNoCC.hostPlatform) ''
```

Comment:
Perhaps we can remove the `optionalString` from `postPatchelf` too. The otherwise empty string seems to cause issues so I think it would be better to always define the phase and only call it if it can be run at all (like you did).

Reply:
Done!